totalRP3/Modules/TooltipSkins/ElvUI.lua

```diff
@@ -108,24 +108,35 @@ TRP3_API.module.registerModule({
 				for _, tooltip in pairs(TOOLTIPS) do
 					if _G[tooltip] then
 						-- We check that the tooltip exists and then add it to ElvUI
-						TT:SecureHookScript(_G[tooltip], 'OnShow', 'SetStyle')
+						TT:SecureHookScript(_G[tooltip], 'OnShow', 'SetStyle');
 					end
 				end
 			end
 
 			-- 9.2 changed Tooltips to use NineSlice but TargetFrame doesn't use it, therefore we need to use the old styling function
 			local function SetStyleForTargetFrame(tt)
-				if not tt or (tt == _G["ElvUI"][1].ScanTooltip or tt.IsEmbedded or not tt.SetTemplate or not tt.SetBackdrop) or tt:IsForbidden() then return end
-				tt.customBackdropAlpha = TT.db.colorAlpha
-				tt:SetTemplate('Transparent')
+				if not tt or (tt == _G["ElvUI"][1].ScanTooltip or tt.IsEmbedded or not tt.SetTemplate or not tt.SetBackdrop) or tt:IsForbidden() then return; end
+				tt.customBackdropAlpha = TT.db.colorAlpha;
+				tt:SetTemplate('Transparent');
 			end
 
 			function skinTargetFrame()
 				-- Go through each skinnable frames from our table
 				for _, frame in pairs(SKINNABLE_FRAMES) do
+					local parentKey;
+
+					-- Check if a skinnable frame includes a parentKey
+					if frame:find("%.") then
+						frame, parentKey = frame:match("(.+)%.(.+)");
+					end
```

Comment:
```suggestion
					local parentKey;
					frame, parentKey = string.split(".", frame, 2);
```

Believe this is functionally equivalent - do test.

Reply:
Confirmed works the same, I'll batch it in.